### PR TITLE
Bug/nav 1.4.4

### PR DESCRIPTION
**Pull request by Evtaft22**
- Fixed issues with routing from page to page using the navbar

### CODE REVIEW

File: client/src/components/PostFeedEntry.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const PostFeedEntry = ({ post, users, tags }) => {
 
   return (
     <div>
-      <Link style={{ color: "#ff8c00" }} to={`/fullMessage/${post.id_user}${post.id}`}>
+      <Link style={{ color: "#ff8c00" }} to={`/home/${post.id_user}${post.id}`}>
```
Comment (jstrzesz) [inline client/src/components/PostFeedEntry.jsx:27]: move styling to css file, no inline styling

File: client/src/components/PostFullMessage.jsx [hunk 2]
```diff
@@ -105,7 +154,7 @@ const PostFullMessage = ({ match, menu, currentUser }) => {
         </div>
         <div style={{ marginTop: "160px" }}>
```
Comment (jstrzesz) [inline client/src/components/PostFullMessage.jsx:155]: move styling to css file, no inline styling

File: client/src/components/PostFullMessage.jsx [hunk 3]
```diff
@@ -105,7 +154,7 @@ const PostFullMessage = ({ match, menu, currentUser }) => {
...
           <div style={{ fontSize: "18px" }}>{post.message}</div>
```
Comment (jstrzesz) [inline client/src/components/PostFullMessage.jsx:156]: move styling to css file, no inline styling

File: client/src/components/PostFullMessage.jsx [hunk 4]
```diff
@@ -105,7 +154,7 @@ const PostFullMessage = ({ match, menu, currentUser }) => {
...
-          <div style={{ fontSize: "16px", marginTop: "10px" }}>{postTags.join("   ")}</div>
+          <div style={{ fontSize: "16px", marginTop: "10px" }}>{postTags.join(" ")}</div>
```
Comment (jstrzesz) [inline client/src/components/PostFullMessage.jsx:157]: move styling to css file, no inline styling

File: client/src/components/PostFullMessage.jsx [hunk 5]
```diff
@@ -139,7 +188,7 @@ const PostFullMessage = ({ match, menu, currentUser }) => {
             type="text"
             placeholder="Message"
           />
-          <button onClick={sendMessage} style={{ marginLeft: "5px", backgroundColor: "#eb8c34" }}>
+          <button onClick={onSubmit} style={{ marginLeft: "5px", backgroundColor: "#eb8c34" }}>
```
Comment (jstrzesz) [inline client/src/components/PostFullMessage.jsx:191]: move styling to css file, no inline styling

**Review by bsbrionesOS — APPROVED** [review]

**Review by HenokSamuel — APPROVED** [review]

**Pull request merged**